Assets/Content/Items/Functional/Tools/Generic/desk lamp.prefab

```diff
@@ -266,21 +266,24 @@ MonoBehaviour:
   m_GameObject: {fileID: 5021521585556056798}
   m_Enabled: 1
   m_EditorHideFlags: 0
-  m_Script: {fileID: 11500000, guid: b8e907cf58c0ce0439d34cd7cf644216, type: 3}
+  m_Script: {fileID: 11500000, guid: 1ae0cc9eb4c1ca6428dc327bb781cdae, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
   syncMode: 0
   syncInterval: 0.1
   ItemId: lamp_desk_grey
   Name: Desk Lamp
   Volume: 15
-  container: {fileID: 0}
   sprite: {fileID: 21300000, guid: 66026a73b54c5814aa62ed061b556bbd, type: 3}
   prefab: {fileID: 5021521585556056798}
   attachmentPoint: {fileID: 3367512477711822723}
+  attachmentPointAlt: {fileID: 0}
   bulkSize: 3
   traits: []
---- !u!114 &705127767441947080
+  Size: {x: 0, y: 0}
```

Comment:
What do these sizes do? Should they be nonzero?

Reply:
They are for the inventory system... we need to iron out a few details with the system still so many items have been given an appropriate size yet.